### PR TITLE
misc: remove type annotations from operand definitions [NFC]

### PR DESCRIPTION
**Pull request by superlopuh**
I regret writing the original sed script to add the type annotations, as they're not necessary.

### CODE REVIEW

File: remove_annotated.sed
Comment (superlopuh) [inline remove_annotated.sed:1]: no longer necessary

**Comment by superlopuh** [comment]
I double-checked all the files and they're fine

**Comment by superlopuh** [comment]
@math-fehr, as you can imagine, this PR accumulates conflicts very quickly, do you think we could time your review somehow? I'm happy to update it whenever you're ready to take a look at it, please ping me.

**Comment by math-fehr** [comment]
Sorry for the delay, let's merge this now

**Review by math-fehr — APPROVED** [review]

**Pull request merged**